docassemble/assemblylinewizard/test_map_names.py

```diff
@@ -85,52 +49,155 @@ def test_mapped_scenarios(self, run_from_yaml=False):
-  "plaintiff": "str(plaintiffs[0])",
-  "defendant": "str(defendants[0])",
-  "petitioner": "str(petitioners[0])",
-  "respondent": "str(respondents[0])",
-  "spouse": "str(spouses[0])",
-  "parent": "str(parents[0])",
-  "guardian": "str(guardians[0])",
-  "caregiver": "str(caregivers[0])",
-  "attorney": "str(attorneys[0])",
-  "translator": "str(translators[0])",
-  "debt_collector": "str(debt_collectors[0])",
-  "creditor": "str(creditors[0])",
-  "court": "str(courts[0])",
-  "other_party": "str(other_parties[0])",
-  "child": "str(children[0])",
-  "guardian_ad_litem": "str(guardians_ad_litem[0])",
-  "witness": "str(witnesses[0])",
-  "users": "str(users)",
-  "plaintiffs": "str(plaintiffs)",
-  "defendants": "str(defendants)",
-  "petitioners": "str(petitioners)",
-  "respondents": "str(respondents)",
-  "spouses": "str(spouses)",
-  "parents": "str(parents)",
-  "guardians": "str(guardians)",
-  "caregivers": "str(caregivers)",
-  "attorneys": "str(attorneys)",
-  "translators": "str(translators)",
-  "debt_collectors": "str(debt_collectors)",
-  "creditors": "str(creditors)",
-  "courts": "str(courts)",
-  "other_parties": "str(other_parties)",
-  "children": "str(children)",
-  "guardians_ad_litem": "str(guardians_ad_litem)",
-  "witnesses": "str(witnesses)",
+  "user": "users[0]",
+  "plaintiff": "plaintiffs[0]",
+  "defendant": "defendants[0]",
+  "petitioner": "petitioners[0]",
+  "respondent": "respondents[0]",
+  "spouse": "spouses[0]",
+  "parent": "parents[0]",
+  "guardian": "guardians[0]",
+  "caregiver": "caregivers[0]",
+  "attorney": "attorneys[0]",
+  "translator": "translators[0]",
+  "debt_collector": "debt_collectors[0]",
+  "creditor": "creditors[0]",
+  "court": "courts[0]",
+  "other_party": "other_parties[0]",
+  "child": "children[0]",
+  "guardian_ad_litem": "guardians_ad_litem[0]",
+  "witness": "witnesses[0]",
+  "users": "users",
+  "plaintiffs": "plaintiffs",
+  "defendants": "defendants",
+  "petitioners": "petitioners",
+  "respondents": "respondents",
+  "spouses": "spouses",
+  "parents": "parents",
+  "guardians": "guardians",
+  "caregivers": "caregivers",
+  "attorneys": "attorneys",
+  "translators": "translators",
+  "debt_collectors": "debt_collectors",
+  "creditors": "creditors",
+  "courts": "courts",
+  "other_parties": "other_parties",
+  "children": "children",
+  "guardians_ad_litem": "guardians_ad_litem",
+  "witnesses": "witnesses",
 
   # Starts with no names
   "docket_number": "docket_numbers[0]",
-  "docket_numbers": "str(docket_numbers)",
+  "docket_numbers": "docket_numbers",
+  "signature_date": "signature_date",
+
+  # Reserved start with unreserved end
+  "user_address_street2_zip": "user_address_street2_zip",  
+
+  # Not reserved
+  "my_user_name_last": "my_user_name_last",
+  "foo": "foo",
+}
+
+interview_order_scenarios = {
+  # Reserved whole words
+  "signature_date": "signature_date",
+  "attorney_of_record_address_on_one_line": "attorney_of_record_address_on_one_line",
```

Comment:
I can't find this in the question code. What file is it in?

Reply:
It's being reserved in [generator_constants.py](https://github.com/SuffolkLITLab/docassemble-assemblylinewizard/blob/297410a3038fbed8f1c0d217ef70bdb3c71d25a9/docassemble/assemblylinewizard/generator_constants.py#L14), but you're right, it doesn't get used anywhere else. I'm not sure what it's main purpose is?

Reply:
I guess we should put it in the question library. I can't see a good reason to not make it available

Reply:
Until we put it in the question library, maybe we can comment it out in the .py file. Unless the PDF checker doesn't work the way I think it works. We can make that a separate issue.

Reply:
That's a good point, I'll comment it back out. I'd offer to make a question for the library, but I don't know what how it's different than `attorneys[0].address.on_one_line()`.